UnitTests/WebTestBase.cs

```diff
@@ -1,84 +0,0 @@
-﻿using Microsoft.AspNetCore.Hosting;

```

Comment:
Jeg så kommentaren din om problemet med integrasjonstesting.

Ta en titt på:
- https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.testing.webapplicationfactory-1?view=aspnetcore-6.0
- https://github.com/dotnet/AspNetCore.Docs.Samples/tree/main/test/integration-tests/IntegrationTestsSample

Jeg håper det er mulig å skrive om `WebTestBase.cs` til å bruke `WebApplicationFactory` fra  `Microsoft.AspNetCore.Mvc.Testing`, og samtidig ha mindre endringer i integrasjonstestklassene som arver `WebTestBase`.

Reply:
Du sendte meg virkelig ned et rabbit hole her altså : )

Se løsningen her: https://github.com/codeape2/netmockery/pull/22/commits/a6341a56174551625803fb56db5fb84d5cd5d19f

Pga `WebApplicationFactory `passet args til `Program.cs Main(args)` i et nytt format "ENVIRONMENTS=blabla" osv, så bestemte jeg meg for å skrive om `CommandLineParser`. Det er fortsatt ikke ideelt, men jeg tenker det er godt nok.